assets/js/theme/common/product-details.js

```diff
@@ -165,7 +165,8 @@ export default class ProductDetails {
             if (view.attr('data-event-type')) {
                 view.attr('data-product-variant', productVariant);
             } else {
-                const productName = view.find('.productView-title')[0].innerText;
+                const productName = view.find('.productView-title')[0].innerText.replace(/"/g, '\\$&');
+                console.log(productName);
```

Comment:
@BC-tymurbiedukhin can we remove the log statement.

Reply:
console.log statement has been removed